src/main/java/jenkins/plugins/openstack/compute/JCloudsCloud.java

```diff
@@ -92,6 +100,10 @@
 
     private /*final*/ @Nonnull String credentialId; // Name differs from property name not to break the persistence
 
+    private @CheckForNull SectionDisabled disabled;
```

Comment:
Correct - it'll be null after migration, and maybe after Web-UI construction if we don't set it explicitly.
Personally, I prefer code to avoid saying `<SectionDisabled />` in the persisted XML - if it's got nothing to say, why not omit it entirely (which is why I'm generally in no hurry to make things non-null) but I agree that making it non-null would simplify the code in this case.
...and if that's done, it would remove the need to call setDisabled in a few places.